tasks/publish-to-mrrc/README.md

```diff
@@ -0,0 +1,12 @@
+# publish-to-mrrc
+
+Tekton task publish the maven artifacts to MRRC(maven.repository.redhat.com) service. MRRC is used to host maven artifacts of Red Hat Middleware products.
+This task will work with [collect-mrrc-task](../collect-mrrc-params/README.md) together to do the MRRC publishment work. It accepts the `mrrc.env` file from the [collect-mrrc-task](../collect-mrrc-params/README.md) and use the variables in it as parameters for the MRRC publishing task. It also uses `cosignPubKeyConfig` as public key to do the maven zip signature and attestation check.
+
+## Parameters
+
+| Name               | Description                                                                                                                                     | Optional | Default value |
+|--------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| mrrcParamFilePath  | Path of the mrrc.env file which contains the MRRC parameters as environment viariables                                                          | No       | -             |
+| cosignPubKeyConfig | The configmap which contains the public key to do the cosign signature and attestation check of maven zip file. Uses "consign.pub" as file key. | No       | -             |
```

Comment:
I know there is a thread in the yaml, but opening one here to remove this line

Reply:
Sure. Will remove.

Reply:
Removed in new commit.